e2e_tests/tests/nightly/test_convergence.py

```diff
@@ -361,3 +364,9 @@ def test_cifar10_byol_pytorch_accuracy() -> None:
             target_accuracy, len(trial_metrics["steps"]), validation_accuracies
         )
     )
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
it's odd to duplicate this code, does it have to be in all files? if yes, can it be imported from the other one instead of repeating the code?